VisualStudio.gitignore

```diff
@@ -21,6 +21,7 @@ bld/
 [Bb]in/
 [Oo]bj/
 [Ll]og/
+[Bb]ld/
```

Comment:
Is this one also Cordova-specific? I'd love to understand more about where this one is coming from...